azure-devops-extension/run-automator-build-task/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "automator-tools-azure-pipelines-extension",
+  "version": "1.0.1",
+  "description": "Azure task for running the build of Automator/Deepcrawl test suite.",
+  "main": "index.js",
+  "scripts": {
+    "build": "ttsc",
+    "start": "node ./dist/index.js",
+    "lint": "yarn prettier --write . && yarn eslint --ext .js,.ts src"
+  },
+  "author": "Andrei Bitca, Daniel Mon Chit & Lukas Kalcok",
+  "license": "ISC",
+  "devDependencies": {
+    "@types/axios": "^0.14.0",
+    "@typescript-eslint/eslint-plugin": "^4.1.1",
+    "@zerollup/ts-transform-paths": "^1.7.18",
+    "eslint": "^7.9.0",
+    "eslint-config-deepcrawl": "^5.6.0",
+    "eslint-plugin-array-func": "^3.1.7",
+    "eslint-plugin-clean-code": "^0.1.12",
+    "eslint-plugin-filenames": "^1.3.2",
+    "eslint-plugin-graphql": "^4.0.0",
+    "eslint-plugin-import": "^2.22.0",
+    "eslint-plugin-jest": "^24.0.2",
+    "eslint-plugin-json-format": "^2.0.1",
+    "eslint-plugin-no-loops": "^0.3.0",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-promise": "^4.2.1",
+    "eslint-plugin-sonarjs": "^0.5.0",
+    "jest": "^26.5.2",
+    "prettier": "^2.1.2",
+    "tfx-cli": "^0.8.3",
```

Comment:
Do we need this here if it is in the package.json in parent directory?

Reply:
Should be remove indeed.